python/prohibition_web_svc/models.py

```diff
@@ -577,12 +580,14 @@ class VIForm(db.Model):
     form_id = db.Column(db.Integer, primary_key=True)
     event_id = db.Column(db.Integer, db.ForeignKey('event.event_id'))
     gender = db.Column(db.String)
+    driver_is_regist_owner = db.Column(db.String)
     driver_licence_expiry = db.Column(db.DateTime)
     driver_licence_class = db.Column(db.String)
     unlicenced_prohibition_number = db.Column(db.String)
     belief_driver_bc_resident = db.Column(db.String)
     out_of_province_dl = db.Column(db.String)
     out_of_province_dl_number = db.Column(db.String)
+    out_of_province_dl_expiry = db.Column(db.String)
```

Comment:
was this column added to the dataclasses up top?

Reply:
added